notebooks/plot_some_color_palettes.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>Can we add a Sesame Street color theme?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>